c3po/api/dto.py

```diff
@@ -0,0 +1,12 @@
+# Data Transfer Object- Responsible for carrying data between processes
+from flask import current_app
+from flask_restx import Namespace, fields
+
+
+class FeedDto:
+    api = Namespace('feed', description='Feed Related operations')
```

Comment:
Is `api` the appropriate name? This is a minor nitpick but the re-use of the `api` term across just this one PR already seems confusing. Would `namespace` or `ns` be more appropriate? And import later as:
```py
from c3po.api.dto import FeedDto

feed_ns = FeedDto.api
feed_song = FeedDto.songObject
```
Feel free to use other names, but giving an example to explain the reason for the comment.

Reply:
We're using `api` inside this module and then when importing inside the Blueprint definition, calling it as feed_ns. I think we did this to maintain consistency in each module about what API refers to. API is confusing here because it's also the folder's name so we'll change it to avoid confusion.